src/main/java/io/confluent/idesidecar/restapi/models/graph/RealLocalFetcher.java

```diff
@@ -76,13 +76,6 @@ public class RealLocalFetcher extends ConfluentLocalRestClient implements LocalF
           String.class
       );
 
-  static final String CONFLUENT_LOCAL_DEFAULT_SCHEMA_REGISTRY_URI = ConfigProvider
-      .getConfig()
-      .getValue(
-          "ide-sidecar.connections.confluent-local.default.schema-registry-uri",
```

Comment:
If this is removed, does anything else have to use https://github.com/confluentinc/ide-sidecar/blob/main/src/main/resources/application.yml#L126?

Reply:
Good point, there's nothing having to use that. Will clean it up once we remove `LOCAL` connections for good.